cpp/lib/src/ZoteroHarvesterConversion.cc

```diff
@@ -246,37 +246,37 @@ bool ZoteroItemMatchesExclusionFilters(const Util::HarvestableItem &download_ite
 void ConvertZoteroItemToMetadataRecord(const std::shared_ptr<JSON::ObjectNode> &zotero_item,
                                        MetadataRecord * const metadata_record)
 {
-    metadata_record->item_type_ = zotero_item->getStringValue("itemType");
-    metadata_record->title_ = zotero_item->getOptionalStringValue("title");
-    metadata_record->short_title_ = zotero_item->getOptionalStringValue("shortTitle");
-    metadata_record->abstract_note_ = zotero_item->getOptionalStringValue("abstractNote");
-    metadata_record->publication_title_ = zotero_item->getOptionalStringValue("publicationTitle");
+    metadata_record->item_type_ = HtmlUtil::StripHtmlTags(zotero_item->getStringValue("itemType"));
```

Comment:
Consider an inline helper that combines HtmlUtil::StripHtmlTags with getStringValue defined just above this function.